src/blockmap.jl

```diff
@@ -48,7 +48,7 @@ function rowcolranges(maps, rows)::Tuple{Vector{UnitRange{Int}},Vector{UnitRange
 end
 
 function Base.size(A::BlockMap)
-    return A.rowranges[end][end], A.colranges[end][end]
+    return last(A.rowranges[end]), last(A.colranges[end])
```

Comment:
Could you please turn this into a one-liner?
```julia
Base.size(A::BlockMap) = (last(A.rowranges[end]), last(A.colranges[end]))
```
This is the format that I have in #66 for this function anyway.